### PR TITLE
[risk=low][RW-13549] Institution Admin Initial Credits Expiration Bypass UI

### PR DESCRIPTION
**Pull request by jmthibault79**
Note: this is not connected to initial credits expiration logic yet.  It only updates a DB value.

credits_bypass_2

---
**PR checklist**

- [x] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have manually tested this change and my testing process is described above.
- [x] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [x] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [x] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: ui/src/app/utils/institutions.tsx [hunk 1]
```diff
@@ -178,17 +178,6 @@ export function updateMembershipRequirement(
   });
 }
 
-export function updateRequireEra(
```
Comment (jmthibault79) [inline ui/src/app/utils/institutions.tsx:181]: unused

File: ui/src/app/pages/admin/institution/admin-institution-edit.spec.tsx [hunk 2]
```diff
@@ -760,4 +793,21 @@ describe('AdminInstitutionEditSpec - add mode', () => {
     ).not.toBeInTheDocument();
     await user.unhover(getAddButton());
   });
+
+  it('Should allow setting bypassInitialCreditsExpiration', async () => {
```
Comment (evrii) [inline ui/src/app/pages/admin/institution/admin-institution-edit.spec.tsx:797]: Is this different than 'Should allow updating bypassInitialCreditsExpiration from false to true'?

File: ui/src/app/pages/admin/institution/admin-institution-edit.tsx [hunk 3]
```diff
@@ -50,15 +46,19 @@ import {
...
 import { NavigationProps } from 'app/utils/navigation';
-import { MatchParams } from 'app/utils/stores';
+import { MatchParams, serverConfigStore } from 'app/utils/stores';
 import { canonicalizeUrl } from 'app/utils/urls';
 import { withNavigation } from 'app/utils/with-navigation-hoc';
 
+import {
+  MembershipRequirements,
+  OrganizationTypeOptions,
+} from './admin-institution-options';
```
[the remark this answers is not in the document]
Comment (evrii) [reply ui/src/app/pages/admin/institution/admin-institution-edit.tsx:60]: No strong preference here, but is the `./` needed? I thought that the linter gives a hard time for relative paths.

File: ui/src/app/pages/admin/user/admin-user-profile.tsx [hunk 4]
```diff
@@ -348,9 +348,8 @@ const ToggleForModule = (props: ToggleProps) => {
       }}
     >
       <CommonToggle
-        name=' '
+        name={`Bypass ${moduleName}`}
```
Comment (evrii) [inline ui/src/app/pages/admin/user/admin-user-profile.tsx:351]: Good catch

**Review by evrii — APPROVED** [review]
Test locally and worked as expected

File: ui/src/app/pages/admin/institution/admin-institution-edit.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (jmthibault79) [reply ui/src/app/pages/admin/institution/admin-institution-edit.tsx:60]: It's not required.  I like to use this style when they are closely related.  The linter seems to complain about `../` but not `./`

File: ui/src/app/pages/admin/institution/admin-institution-edit.spec.tsx (hunk 2 above)
Comment (jmthibault79) [reply ui/src/app/pages/admin/institution/admin-institution-edit.spec.tsx:797]: this is the "add new Inst" case and the other is "edit existing Inst"

File: ui/src/app/pages/admin/user/admin-user-profile.tsx (hunk 4 above)
Comment (jmthibault79) [reply ui/src/app/pages/admin/user/admin-user-profile.tsx:351]: Oops.  #8779

**Pull request merged**